### PR TITLE
Contest Submission

### PR DESCRIPTION
**Pull request by miguelrodrigues22**
On the contest detail (https://github.com/runtimerevolution/revent-app/issues/66) we should add a way for a user to submit a picture to the contest.

### CODE REVIEW

File: components/Submissions/SubmissionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      const contestSubmission: ContestSubmissionInput = {
+        contest: contestID,
+        picture,
+      }
+      console.log('contestSubmission', contestSubmission)
+
+      const response = await createContestSubmission({
+        variables: { contestSubmission },
+      })
+      console.log(response.data.create_contestSubmission)
```
Comment (JdFSilva) [inline components/Submissions/SubmissionForm.tsx:66]: remove logs

File: components/Submissions/SubmissionForm.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  votes?: string[]
+}
+
+interface PictureInput {
+  user: string
+  picture_path: string
+  likes?: string[]
+}
+
+export default function SubmissionForm({ contestID, setShowAddPhotoForm }) {
```
Comment (JdFSilva) [inline components/Submissions/SubmissionForm.tsx:24]: add props interface

File: components/Submissions/SubmissionForm.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  picture: string
+  submission_date?: string
+  votes?: string[]
+}
+
+interface PictureInput {
+  user: string
+  picture_path: string
+  likes?: string[]
+}
```
Comment (JdFSilva) [inline components/Submissions/SubmissionForm.tsx:22]: this is unused

**Review by JdFSilva — CHANGES_REQUESTED** [review]

File: components/ErrorMessage.tsx [hunk 4]
```diff
@@ -1,9 +1,5 @@
 import React from 'react'
 
-interface ErrorMessageProps {
-  error: string
-}
-
-export default function ErrorMessage({ error }: ErrorMessageProps) {
+export default function ErrorMessage({ error }) {
```
[the remark this answers is not in the document]
Comment (miguelrodrigues22) [reply components/ErrorMessage.tsx:3]: done

File: components/Submissions/SubmissionForm.tsx (hunk 1 above)
Comment (miguelrodrigues22) [reply components/Submissions/SubmissionForm.tsx:66]: done

File: components/Submissions/SubmissionForm.tsx (hunk 3 above)
Comment (miguelrodrigues22) [reply components/Submissions/SubmissionForm.tsx:22]: removed

File: components/Submissions/SubmissionForm.tsx (hunk 2 above)
Comment (miguelrodrigues22) [reply components/Submissions/SubmissionForm.tsx:24]: done

Comment (JdFSilva) [reply components/Submissions/SubmissionForm.tsx:18]: not done, please make sure to apply the changes before saying done

**Review by JdFSilva — CHANGES_REQUESTED** [review]

File: components/Submissions/SubmissionForm.tsx (hunk 2 above)
Comment (miguelrodrigues22) [reply components/Submissions/SubmissionForm.tsx:18]: I'm so sorry, I must have missed it, properly done now

**Review by FilipaSdS — APPROVED** [review]

**Pull request merged**